build.sbt

```diff
@@ -57,9 +58,10 @@ lazy val pekkoPlugin = project
     addSbtPlugin(
       // When updating the sbt-paradox version,
       // remember to also update project/plugins.sbt
-      "com.lightbend.paradox" % "sbt-paradox" % "0.10.3"),
-    addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "1.0.0"),
-    addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-project-info" % "3.0.0"),
+      "com.lightbend.paradox" % "sbt-paradox" % "0.9.2"),
```

Comment:
Is there a reason you downgraded the dependencies? I suspect this may be due to pekko-core however there is a workaround for this (see https://github.com/apache/incubator-pekko/issues/92#issue-1523977563)

Reply:
I can try if it works but it's of course not ideal since there is no compatibility guarantee that would ensure that a plugin compiled against the latest version would work against older versions.

Reply:
So I did try this locally with the incubator-pekko-sbt-paradox and it worked (but that is without the theme). The reason why I am mentioning this is that we do have the problem with projects outside of pekko-core not needing JDK 1.8 and since pekko-core is the outlier here I think it makes sense to use the latest dependencies for the plugin.

Reply:
> that we do have the problem with projects outside of pekko-core not needing JDK 1.8 (

That's true but also not a problem since 1.8 artifacts would work anywhere...

> pekko-core is the outlier here I think it makes sense to use the latest dependencies for the plugin.

I changed it now to the latest but like this there will always be the risk of changing something here and it will not work in pekko core because of incompatibilities...